typos.toml

```diff
@@ -22,6 +22,7 @@ extend-ignore-re = [
 
 [type.rust.extend-words]
 nd = "nd"
+typ = "typ"
```

Comment:
Can you explain this line?

Reply:
Added a new mapping "typ" to the extend-word list for rust typos, fixes https://github.com/nushell/nushell/pull/15418

Reply:
Ah in